modeltrans/manager.py

```diff
@@ -79,13 +79,12 @@ def _add_i18n_annotation(
         version of a field from the jsonb field to allow filtering and ordering.
 
         Arguments:
-            field (TranslatedVirtualField): the virtual field to create an annotation for.
+            virtual_field (TranslatedVirtualField): the virtual field to create an annotation for.
+            fallback (bool): If `True`, `COALESCE` will be used to get the value of the original
```

Comment:
Because it is a boolean, I would expect this argument to be called something like `use_fallback`.

Reply:
yes, but not something to change now.